neat/tools/fittools/compartmentfitter.py

```diff
@@ -720,42 +741,24 @@ def fitCapacitance(self, inds=[0], check_fit=True, force_tau_m_fit=False,
             capacitances are set to mach membrane time scale
         force_tau_m_fit: bool (optional, default ``False``)
             force capacitance fit through membrance time scale matching
+        use_all_channels_for_passive: bool (optional, default ``True``)
```

Comment:
this is not a keyword argument for this function

Reply:
Removed the item